docs/unc_longleaf.md

```diff
@@ -0,0 +1,22 @@
+# nf-core/configs: Longleaf High-Performance Computing Cluster, University of North Carolina at Chapel Hill
+
+> **NB:** You will need an [account](https://help.rc.unc.edu/getting-started-on-longleaf/) to use the HPC cluster to run the pipeline.
+
+We have configured the compute clusters to Apptainer (Singularity) loaded by default. Do not load the Singularity module or it will fail, as it is an older version than what's on the compute nodes.
```

Comment:
Is it correct that you use the `apptainer` scope in the profile (e.g. you are running an older vresion of nextflow by default, or running older pipelines?)

Reply:
sorry, not sure I follow?

Reply:
I was being dense - i changed singularity to apptainer. Thanks!

Reply:
Sorry, put the comment in the wrong place, you got it though 👍